README.md

```diff
@@ -65,22 +65,6 @@ The default port is 1337, to use a different port set the PORT environment varia
 
```

Comment:
the npm start thing works when cloning the repo, and running from it, we may not wanna advertise that either